examples/contracts/cw1-subkeys/src/cw1.rs

```diff
@@ -7,6 +7,9 @@ use crate::error::ContractError;
 
 impl Cw1 for Cw1SubkeysContract<'_> {
```

Comment:
Just to confirm - the contract should be generic over exec and query, but I assume there would be separate PR for that, right?

Reply:
Yes, this PR only deals with interfaces